modular_skyrat/master_files/code/modules/client/preferences_savefile.dm

```diff
@@ -13,6 +13,7 @@
 #define VERSION_UNDERSHIRT_BRA_SPLIT 4
 #define VERSION_CHRONOLOGICAL_AGE 5
 #define VERSION_LANGUAGES 6
+#define VERSION_LOADOUT_PRESETS 7
```

Comment:
Missing undef, and apparently the other two above it miss it too (That's my mistakes coming to bite back).